src/dotnet-toolbox/UninstallCommand.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using DotNetToolbox.Helpers;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox
+{
+    public class UninstallCommand : CommandLineApplication
+    {
+        private ToolboxConfiguration _toolboxConfig;
+        public UninstallCommand(CommandLineApplication parent)
+        {
+            Parent = parent;
+            Name = "uninstall";
+            PackageArgument = new PackageArgument(this);
+            Arguments.Add(PackageArgument);
+            OnExecute((Func<Task<int>>)Run);
+            Parent.Commands.Add(this);
+            HelpOption("-h|--help");
+            _toolboxConfig = new ToolboxConfiguration();
+
+        }
+        public PackageArgument PackageArgument { get; set; }
+
+        public async Task<int> Run()
+        {
+            var pkg = new PackageMetadata(PackageArgument.Value);
```

Comment:
It is an object we use all over the place. I don't see a problem in using it here.